src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -222,6 +222,12 @@ internal sealed class Section
             /// </summary>
             public static StringComparison NameComparer { get; } = StringComparison.Ordinal;
 
+            /// <summary>
+            /// Used to compare <see cref="Name"/>s of sections. Specified by editorconfig to
+            /// be a case-sensitive comparison.
+            /// </summary>
+            public static IEqualityComparer<string> NameEqualityComparer { get; } = StringComparer.Ordinal;
```

Comment:
Above is a `StringComparison`, this is a `StringComparer`. Why the difference? And these comments are the same. Seems like duplication?

Reply:
`StringComparison` is passed directly to `string.Compare` vs `StringComparer` which goes to dictionaries etc. I was thinking it being public meant I couldn't remove it, but it's actually on an internal class anyway, so I'll clean it up.

---
In reply to: [423390383](https://github.com/dotnet/roslyn/pull/43889#discussion_r423390383) [](ancestors = 423390383)